core/trino-main/src/main/java/io/trino/FeaturesConfig.java

```diff
@@ -521,4 +522,17 @@ public FeaturesConfig setAllowSetViewAuthorization(boolean allowSetViewAuthoriza
         this.allowSetViewAuthorization = allowSetViewAuthorization;
         return this;
     }
+
+    public boolean isAcceleratedRepartitioningEnabled()
+    {
+        return acceleratedRepartitioningEnabled;
+    }
+
+    @Config("accelerated-repartitioning.enabled")
```

Comment:
I'm not sure if we really need the kill switch, but if we're going to have it let's rename `accelerated` -> `optimized` or `columnar` and provide a more useful config description.

Reply:
Also, we should plan to remove the switch and the legacy implementation in a few releases.

Reply:
The row-wise partitioning implementation is still used by `PagePartitioner#partitionPage` under certain conditions where columnar partitioning might be slower. So we'll only be able to remove flag in the future.

Reply:
the primary question is then. Do we want/need a kill switch for the `PagePartitioner` columnar approach?
I now lean towards we don't. @sopel39 WDYT?

Reply:
I think we can skip it. It would just be a safety valve in case of a problem with release